resources/render_hero.py

```
@@ -0,0 +1,93 @@
+svg { display: block; width: 100vw; height: 100vh; }
+</style></head><body>%s</body></html>"""
+
+
+def main() -> int:
+    if not SOURCE.exists():
+        print(f"Error: {SOURCE} not found")
+        return 1
+
+    svg_markup = SOURCE.read_text(encoding="utf-8")
+    try:
+        root = ET.fromstring(svg_markup)
```

Ruff (0.16.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)